runtime/src/NativeModules/Amplitude.native.tsx

```diff
@@ -1,3 +1,11 @@
-import * as Amplitude from 'expo-analytics-amplitude';
+// Disable Amplitude for SDK 46
```

Comment:
ah - i didn't realize that we were using this in the snack runtime 😰

Reply:
I already pinged @tcdavis, but we aren't using analytics from the Runtime anymore. So should be good :)